### PR TITLE
Added link_to_related_model helper

### PR DESCRIPTION
**Pull request by doganmeh**
I plan on using this helper a lot, so might others need.

### CODE REVIEW

**Comment by doganmeh** [comment]
Above commit will fix the paging bug. Seems [datatables parameter conversion chart](https://datatables.net/upgrade/1.10-convert) is not up to date, still shows:
    'iDisplayStart' => 'displayStart'
    'iDisplayLength' => 'pageLength'
which does not work, working params are in the commit.

**Comment by jangeador** [comment]
@doganmeh does link to related model only work for one-to-one fields? How will it work with `ManyToOne` or `ManyToMany`?

**Comment by doganmeh** [comment]
It was designed with ForeignKey fields in mind, but it should work with OneToOne fields as well. In summary, related object needs to be a single one. It would not work with ManytoMany..

**Comment by tiliv** [comment]
I'll be pulling this in shortly, but I wanted to give a heads up that I'll probably want to adapt the helper.  As it is right now, the kwarg isn't required, which leads to it running a lookup via ``getattr(obj, None)``.

What this could use is an application of the ``@keyedhelper`` decorator I built, which could enable syntax such as this:
```python
content = columns.TextColumn('content', processor=link_to_related_model('fkname'))
```

This related lookup strategy was neglected because it was it was technically always possible (the example shown in the helper's documentation addresses this) via the ``key`` kwarg.

For example:
```python
helpers.link_to_model(key=helpers.attrgetter('fkname'))
```
or
```python
helpers.link_to_model(key=lambda instance: instance.fkname)
```

Both are wordy, however, so I support the addition of a shorter syntax.

**Comment by doganmeh** [comment]
feel free..

**Comment by doganmeh** [comment]
the use cases of the keyed_helper was not clear to me. Going through the code, I now realize it supports dot notation. So that is probably what was meant by:

> With the help of a ``sort``-style ``key`` argument, the helper can receive all the information..

Thus, the link_to_related_model helper was redundant other than a little syntactic sugar.

File: datatableview/utils.py [hunk 1]
```diff
@@ -24,8 +27,8 @@
 # Since it's rather painful to deal with the datatables.js naming scheme in Python, this map changes
 # the Pythonic names to the javascript ones in the GET request
 OPTION_NAME_MAP = {
```
Comment (devictr) [inline datatableview/utils.py]: This causes the following code in datatables.py line 397 to fail:
```
def _get_current_page(self):
        """
        If page_length is specified in the options or AJAX request, the result list is shorten
ed to
        the correct offset and length.  Paged or not, the finalized object_list is then return
ed.
        """

        # Narrow the results to the appropriate page length for serialization
        if self.config['page_length'] != -1:
            i_begin = self.config['start_offset']
            i_end = self.config['start_offset'] + self.config['page_length']
            object_list = self._records[i_begin:i_end]

        return object_list
```

**Comment by jangeador** [comment]
What is the status on this PR? Will it get pulled in?

**Comment by tiliv** [comment]
Closing this in favor of 553ef52 from #166 — ``link_to_model(attr='fkfield')``

I'll get the paging issues corrected in a separate ticket.

**Pull request closed without merging**